Formula/rmw.rb

```diff
@@ -0,0 +1,30 @@
+class Rmw < Formula
+  desc "Safe-remove utility for the command-line"
+  homepage "https://remove-to-waste.info/"
+  url "https://github.com/theimpossibleastronaut/rmw/releases/download/v0.7.09/rmw-0.7.09.tar.gz"
+  sha256 "d0a85944d03e7ec2a94b25d6d6ac92880fd0d3f63d90bb9ed56b16418fd41c69"
+  license "GPL-3.0-or-later"
+  head "https://github.com/theimpossibleastronaut/rmw.git"
+
+  depends_on "ncurses"
+
+  def install
+    ENV.append_path "PKG_CONFIG_PATH", Formula["ncurses"].opt_lib/"pkgconfig"
```

Comment:
What happens when you build without this? `ncurses`' `pkgconfig` directory should already be in `PKG_CONFIG_PATH` without doing this.

Reply:
I can't try directly as I don't have a macOS computer. But as I noted in my last comment, the build has always failed for me on Travis and more recently, using GitHub actions.

Reply:
> What happens when you build without this? `ncurses`' `pkgconfig` directory should already be in `PKG_CONFIG_PATH` without doing this.

Here's an example of it working when PKG_CONFIG_PATH is used before configure, but then not for 'make distcheck'

https://github.com/theimpossibleastronaut/rmw/runs/2579079173?check_suite_focus=true

And here, when it's added, it completes successfully.
https://github.com/theimpossibleastronaut/rmw/runs/2579116775?check_suite_focus=true

Reply:
You are running that outside of `brew`, though, so you don't benefit from `brew` setting env variables correctly for the build. This build is running inside of `brew`, so `brew` should be setting these env variables.

Reply:
Ok, I've removed it.